pages/trading/other_limits.md

```diff
@@ -2,15 +2,15 @@
 
 Subaccounts may have a limited number of open orders at any one time determined by the net collateral of the subaccount as per:
```

Comment:
suggestion: The net collateral of a subaccount determines the number of open orders it is allowed. See below for limits: